CONTRIBUTING.md

```diff
@@ -49,11 +49,29 @@ Early reviews of the community are not meant as an offense but to give feedback.
 
 A good principle: "Work together, share ideas, teach others."
 
-### Test the book locally before submitting
+## Test the book locally before submitting
 
 Before submitting the PR launch the commands `mdbook build` to make sure that the book builds and `mdbook test` to make sure that
 code examples are correct.
 
+### Markdown lint
+
+To make sure the files comply with our Markdown style we use [markdownlint-cli](https://github.com/igorshubovych/markdownlint-cli). To spare you some manual work to get through the CI test you can use the following commands to automatically fix most of the emerging problems when writing Markdown files.
+
+- Install:
+
+  ```sh
+  npm install -g markdownlint-cli
+  ```
+
+- Check all markdown files:
+  - unix: `markdownlint '**/*.md'`
+  - windows: `markdownlint **/*.md`
+
+- Automatically fix basic errors:
+  - unix: `markdownlint -f '**/*.md'`
+  - windows: `markdownlint -f **/*.md`
+
 ### Important Note
```

Comment:
Can we move up the `important note` to the PR section then?

Reply:
I moved all this "test" section away from the "Creating a PR" section in commit 0182425 because I think it wasn't that related.
In this way the contributing guidelines have a sort of "chronological order"

Reply:
i meant important note is `###` which is under `Check you the article locally` put important note is regarding PRs

Reply:
Yes, now it should be under "Creating a Pull Request"